_chapters/chapter_02.md

```diff
@@ -238,6 +263,14 @@ layout: rails_tutorial
 {% endhighlight %}
 {% endsteps %}
 
+{% aside %}
+### This is what I like to call "error driven development" 😉
+
+One of the things that's great about the Rails framework is that it generally gives the programmer really helpful errors. 
+
+These errors can seem intimidating at first - but as you learn more about the framework and the programming language, they become valuble tools for building your application. So - let's embrace and explore these errors together! 
+{% endaside %}
+
```

Comment:
I wonder if we need this aside. I really like it, but as you work through the tutorial "error driven development" is kinda drilled into your head. (It's been drilled into my head  anyways 😝)